test/js-native-api/test_string/test_string.c

```diff
@@ -310,6 +310,93 @@ static napi_value TestPropertyKeyUtf16AutoLength(napi_env env,
                          auto_length);
 }
 
+static napi_value TestPropertyKeyUtf8(napi_env env, napi_callback_info info) {
```

Comment:
The new test methods must be exposed to JS in the end of this code file and then used from JS test code.